src/components/HeatMap/Row/Row.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Cell from '../Cell';
+import { Wrapper, RowName, Text } from './Row.styles';
+
+
+export default function Row({
+  data, dayLabel, row, getActiveCell, activeCell,
+}) {
+  return (
+    <Wrapper>
+      <RowName>
+        <Text>
+          {dayLabel}
+        </Text>
+      </RowName>
+      {
+        data.map((item, i) => (
+          <Cell
+            key={`${row}${i * 2}`}
+            itemsNumber={item.length}
+            column={i}
+            row={row}
+            activeCell={activeCell}
+            getActiveCell={getActiveCell}
+          />
+        ))
+      }
+    </Wrapper>
+  );
+}
+
+
+Row.propTypes = {
+  data: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
It would be better to use a well-defined prop type instead of `any`. If it's an array of posts you can define a posts prop type somewhere and reuse it throughout the components

Reply:
Ok, object also works :smile: What I meant was defining the exact structure with `shape`. You can create a file that contains custom prop types like

```
const postPropType = shape({
  id: string.isRequired,
  title: string.isRequired,
  ...
})
```

Not sure what kind of fields it needs. But this way it's clear what the data looks like when another developer (or yourself after a year) have a look at the component without running the app and going into the debugger.

But if it's too much effort at this moment it's ok to keep it like is.

Reply:
Yes, I thought I could do it in the next Task when I know which fields I need and then I can eventually refactor the data object.

Reply:
It is actually the whole subreddit info as it is

Reply:
Fair enough :smile: